spec/systems/pagination_spec.rb

```diff
@@ -0,0 +1,119 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'pagination', type: :system do
+  ActiveJob::Base.queue_adapter = :test
```

Comment:
🤔

Reply:
Fixed in 1a828031b2751651923d865f6120b402ecc7dd5b